src/chrome/content/rules/Sydsvenskan.se.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+	Nonfunctional hosts in *.sydsvenskan.se:
+
+		- ^ ᵗ
+		- prenumerera.sydsvenskan.se ᵐ
+		- stjarnklubb.sydsvenskan.se ʳ
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+
+	Mixed content:
+
+		- blogg.sydsvenskan.se
```

Comment:
Sorry, I'm not certain what the difference is. Looking at the HTML code for the site it insists on using http for links within blogg.sydsvenskan.se and for img sources.